CHANGELOG.md

```diff
@@ -8,18 +8,22 @@
   * Add `PayPalVaultDelegate` to handle results from vault flow
   * Add `PayPalWebCheckoutClientError.paypalVaultResponseError` for missing or invalid response from vaulting
 * PaymentButtons
+  * Add new `PaymentButtonSize` case `.regular`
   * Add `custom` case for `PaymentButtonEdges`
   * Support VoiceOver by adding button accessibility labels
   * Add new `PaymentButtonSize` case `.miniWithWordmark`
   * Add new `PayPalButton.Labels` cases: `.addMoneyWith`, `.bookWith`, `.buyWith`, `.buyNowWith`,
   `.checkoutWith`, `.continueWith`, `.contributeWith`, `.orderWith`, `.payLaterWith`, `.payWith`,
   `.reloadWith`, `.rentWith`, `.reserveWith`, `.subscribeWith`, `.supportWith`, `.tipWith`,
   `.topUpWith`
+  * `PayPalButtonSize` `.mini` changed to rectangular button to meet brand guidelines
 * Breaking Changes
   * PaymentButtons
     * Remove `PayPalButton.Labels` cases: `.checkout`, `.buyNow`, `.payLater`
     * Remove `PaymentButtonColor` `.black`, `.silver`, `.blue`, and `.darkBlue`
     * Add `PaymentButtonColor.gold` for `PayPalCreditButton`
+    * Remove `PayPalButton.Labels` cases: `.checkout`, `.buyNow`, `.payLater`
```

Comment:
nit: i think this may be a duplicate of line 22

Reply:
Thanks!